packages/components/src/font-size-picker/index.tsx

```diff
@@ -298,42 +304,45 @@ const UnforwardedFontSizePicker = (
 								</FlexItem>
 								{ withReset && (
 									<FlexItem isBlock>
-										<Button
+										<ResetButton
 											className="components-color-palette__clear"
 											disabled={ value === undefined }
 											onClick={ () => {
 												onChange?.( undefined );
 											} }
 											isSmall
 											variant="secondary"
+											size={ size }
 										>
 											{ __( 'Reset' ) }
-										</Button>
+										</ResetButton>
 									</FlexItem>
 								) }
 							</Flex>
 						) }
-				</div>
-				{ withSlider && (
-					<RangeControl
-						__nextHasNoMarginBottom={ __nextHasNoMarginBottom }
-						className={ `${ baseClassName }__custom-input` }
-						label={ __( 'Custom Size' ) }
-						value={
-							isPixelValue && noUnitsValue
-								? Number( noUnitsValue )
-								: undefined
-						}
-						initialPosition={ fallbackFontSize }
-						onChange={ ( newValue ) => {
-							onChange?.( hasUnits ? newValue + 'px' : newValue );
-						} }
-						min={ 12 }
-						max={ 100 }
-					/>
-				) }
-			</MaybeVStack>
-		</fieldset>
+					{ withSlider && (
+						<RangeControl
+							__nextHasNoMarginBottom
```

Comment:
Is there a reason why `__nextHasNoMarginBottom` is not being passed to `RangeControl` ?

```suggestion
							__nextHasNoMarginBottom={ __nextHasNoMarginBottom }
```

Same applies for the `ToggleGroupControl` component a few lines above.